src/components/reusables/Button/Button.jsx

```diff
@@ -0,0 +1,20 @@
+// react libraries
+import React, { Component } from 'react';
+
+import { Link } from 'react-router-dom';
+
+/**
+ * desc renders Button
+*/
+
```

Comment:
#### Nitpick
Remove this blank line since the comment above is meant for the declared component below.